tests/foreman/api/test_docker.py

```diff
@@ -453,9 +452,31 @@ def test_add_multiple_docker_repos_to_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        upstream_name = u'busybox'
+        product = entities.Product(organization=self.org).create()
+        repos_amount = randint(1, 5)
```

Comment:
You can drop `repos_amount` and place this `randint(1, 5)` statement in to the `repos = [...]` list comprehension. Later on, when `repos_amount` is used, you can use `len(repos)` instead.